soot-infoflow-summaries/summariesManual/com.mashape.unirest.request.GetRequest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<summary fileFormatVersion="101">
+	<methods>
+		<method id="com.mashape.unirest.request.GetRequest header(java.lang.String,java.lang.String)">
+			<flows>
+				<flow isAlias="false" typeChecking="false">
+					<from sourceSinkType="Field"
+						AccessPath="[com.mashape.unirest.request.GetRequest: byte[] bytes]"
```

Comment:
du solltest besser das Feld in der Basisklasse referenzieren, da du sonst mehrere Felder hast und potentiell das falsche nimmst (Field Hiding).

Reply:
habe es angepasst (Soweit ich das verstanden habe war nur bei GetRequest und HttpRequest das Problem sonst wurde ja nichts abgeleitet oder?